### PR TITLE
FS-2119: Redirect after comment creation

### PR DESCRIPTION
**Pull request by tferns**
- After submitting a comment, we return a redirect to the route so it re-fetches the comments

### CODE REVIEW

File: app/assess/routes.py [hunk 1]
```diff
@@ -152,6 +152,15 @@ def display_sub_criteria(
...
 
+            return redirect(
+                url_for(
+                    "assess_bp.display_sub_criteria",
+                    application_id=application_id,
+                    sub_criteria_id=sub_criteria_id,
+                    theme_id=theme_id,
+                )
+            )
+
```
Comment (gio-karunakaran) [inline app/assess/routes.py:163]: 🙌🏾

**Review by gio-karunakaran — APPROVED** [review]
LGTM

**Comment by gidsg** [comment]
@tferns this would be a good use-case for a flash message in addition to the redirect https://flask.palletsprojects.com/en/2.2.x/patterns/flashing/

**Comment by tferns** [comment]
> @tferns this would be a good use-case for a flash message in addition to the redirect https://flask.palletsprojects.com/en/2.2.x/patterns/flashing/

Oh good idea! I know there was some talk of showing feedback when a user had commented so I'll see if I can fish out the story later today and whip something up

**Comment by gidsg** [comment]
@tferns we probably should also redirect to a comments area anchor, else it may not be obvious to the user why they've been redirected, especially without any flash message.

**Comment by tferns** [comment]
> @tferns this would be a good use-case for a flash message in addition to the redirect https://flask.palletsprojects.com/en/2.2.x/patterns/flashing/

@gidsg  I'd implemented these changes here: https://github.com/communitiesuk/funding-service-design-assessment/pull/124

Although Peter has asked design and confirmed no longer necessary, so have closed the PR

**Pull request merged**